src/coprocessor/dag/expr/builtin_time.rs

```diff
@@ -5,14 +5,15 @@ use std::borrow::Cow;
 use chrono::offset::TimeZone;
 use chrono::Datelike;
 
-use super::{EvalContext, Result, ScalarFunc};
 use crate::coprocessor::codec::error::Error;
 use crate::coprocessor::codec::mysql::time::extension::DateTimeExtension;
 use crate::coprocessor::codec::mysql::time::weekmode::WeekMode;
 use crate::coprocessor::codec::mysql::{Duration as MyDuration, Time, TimeType};
 use crate::coprocessor::codec::Datum;
 use crate::coprocessor::dag::expr::SqlMode;
 
+use super::{EvalContext, Result, ScalarFunc};
```

Comment:
Sounds like you're using IntelliJ-Rust's "Optimize Imports"?

Reply:
yes